pkg/azureutils/azure_disk_utils_test.go

```diff
@@ -180,92 +179,6 @@ func TestGetCachingMode(t *testing.T) {
 	}
 }
 
-func TestGetKubeConfig(t *testing.T) {
```

Comment:
since `GetKubeConfig` is removed, can you modify existing ut for `GetKubeClient`?

Reply:
Existing scenarios are covered in TestGetCloudProvider